x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/pipeline_processors_editor_item/pipeline_processors_editor_item.scss

```diff
@@ -3,6 +3,7 @@
 .pipelineProcessorsEditor__item {
   transition: border-color 1s;
```

Comment:
We have `$euiAnimSpeedExtraSlow` along with `$euiAnimSlightResistance` that might work here instead of the hard values.